src/components/styles/Button.styled.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+export const BaseButton = styled.div<{ fontSize?: string }>`
+  padding: 0.5rem 1rem;
+  .ant-btn-text {
+    height: auto;
+    font-size: ${(props) => props.fontSize};
+    border: 0px;
+    border-bottom: transparent 2px solid;
+    background-color: #000;
+    color: #fff;
```

Comment:
Should probably use the given Theme to get the colors in case we decide to use a different shade of white or different themes in the future, will make it easier to implement in the future.
```suggestion
    background-color: ${(props) => props.theme.palette.common.black};
    color: ${(props) => props.theme.palette.common.white}
```

Reply:
yes pls update!

Reply:
dd62687540dc1b09686596594c98118730cff4a2